hasher-matcher-actioner/hmalib/common/models/tests/test_bank_member_with_key.py

```diff
@@ -14,41 +17,100 @@
 
 class BankMemberWithKeyTestCase(BanksTableTestBase, unittest.TestCase):
     def _create_bank(self) -> str:
+        """Create a bank with a unique id."""
+        if not hasattr(self, "_id_count"):
+            self._id_count = 0
```

Comment:
Any reason not to do this in an `__init__()` with a super() call to the parent? hasatter and dynamic attribute modification always looks suspicious to me

Reply:
I prefer declaration and usage of a variable be local (in the sense that they inhabit close line numbers) if it is only used in a single place. 

Can this be in init? Yes. Is any other function using it? Nope.

Reply:
I'm not sure if this usage is truly local if it's being attached to the class variable. Dynamic attribute setting is a source for difficult to debug errors, and so recommend it's usage in code unless absolutely necessary.